examples/seismic/tutorials/16_variable_z_topography.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.    src.coordinates.data[0, -1] = 3*grid.extent[1]/4  # Set depth
</pre><p>Maybe you can explcitly set the depth here, instead of having (3/4)*</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1900/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
I did it this way since the source is vertically positioned in the iteration space, rather than the physical space. Eventually it would be nice for sources to respect coordinate transforms so that one can position them in physical space on a curvilinear grid however